docs/framework/configure-apps/file-schema/winforms/index.md

```diff
@@ -44,7 +44,7 @@ Element  |Description |
 
 Element  |Description |
 ---------|---------|
-[`<configuration>`](Configuration%20File%20Schema%20for%20the%20.NET%20Framework.md) | The root element in every configuration file used by the common language runtime and Windows Forms applications |
+`<configuration>` | The root element in every configuration file used by the common language runtime and Windows Forms applications |
```

Comment:
It would be good if this could be a link. [`<configuration>`](../configuration-element.md) should work.